hedgehog/src/Hedgehog.hs

```diff
@@ -110,7 +110,10 @@ module Hedgehog (
   , eval
   , evalM
   , evalIO
+  , evalMaybe
+  , evalMaybeM
   , evalEither
+  , evalEitherM
```

Comment:
It'd be nice to have those in alphabetical order, after `evalEitherXyz`.

Reply:
sorted evalMaybeXyz after evalEither